samples/socket-api/README.md

```diff
@@ -89,10 +89,44 @@ $ ./iwasm ./socket_opts.wasm
 
 The output is:
 ```bash
-[Client] Create socket
-recv_timeout is expected
-send_timeout is expected
-[Client] Close socket
+[Client] Create TCP socket
+[Client] Create UDP socket
+[Client] Create UDP IPv6 socket
+SO_RCVTIMEO tv_sec is expected
```

Comment:
Not sure if we want to keep maintaining the list? Could we just explain what the output is but not pasting it here? It gets a bit long and might actually make the documentation less readable.